drivers/flash/flash_rpi_pico.c

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2022 Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <errno.h>
```

Comment:
is this required explicitly? Not 100% sure but I guess it should be included in some other headers that are already included in this file

Reply:
Probably not, but I prefer not to be dependent on one of those headers which might be removed in the future.